app/uk/gov/hmrc/minorentityidentificationfrontend/controllers/overseasControllers/CheckYourAnswersController.scala

```diff
@@ -23,7 +23,7 @@ import uk.gov.hmrc.http.InternalServerException
 import uk.gov.hmrc.minorentityidentificationfrontend.config.AppConfig
 import uk.gov.hmrc.minorentityidentificationfrontend.controllers.overseasControllers.{routes => overseasControllerRoutes}
```

Comment:
I think here we can delete "routes" import. The variable "routes" will then default to the overseasControllers package.